product-service/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "product-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "deploy": "serverless deploy -s dev -r eu-west-1 -v",
```

Comment:
You can move region into serverless.ts  file (provider)

Reply:
Moved to `serverless.ts` and cleaned up `npm run deploy` script